recipes/ftfy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ftfy" %}
+{% set version = "4.3.1" %}
+{% set sha256sum = "60af7f4566a966fcb9681f3c93814017e981f5bbfffc2f5f33aadf8238101f4b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    # Setuptools is required for pkg_resources.
+    - setuptools
+    - html5lib
+    - wcwidth
+
+test:
+  imports:
+    - ftfy
+
+  commands:
+    - ftfy -h
+
+about:
+  home: https://github.com/LuminosoInsight/python-ftfy/
+  license: MIT
```

Comment:
Can you please raise an issue or PR with upstream to add the license file to the `MANIFEST.in`?